src/compass/s1_rdr2geo.py

```diff
@@ -103,37 +111,40 @@ def run(cfg, burst=None, save_in_scratch=False):
                               extraiter=rdr2geo_cfg.extraiter,
                               lines_per_block=rdr2geo_cfg.lines_per_block)
 
-        # prepare output rasters
-        topo_output = {'x': (rdr2geo_cfg.compute_longitude, gdal.GDT_Float64),
-                       'y': (rdr2geo_cfg.compute_latitude, gdal.GDT_Float64),
-                       'z': (rdr2geo_cfg.compute_height, gdal.GDT_Float64),
-                       'layover_shadow_mask': (
+        # Dict containing the rdr2geo layers to generate and their filenames
+        # key: rdr2geo layer name
+        # value: (boolean flag; True if layers needs to be generated, layer name)
+        topo_output = {file_name_x: (rdr2geo_cfg.compute_longitude, gdal.GDT_Float64),
+                       file_name_y: (rdr2geo_cfg.compute_latitude, gdal.GDT_Float64),
+                       file_name_z: (rdr2geo_cfg.compute_height, gdal.GDT_Float64),
+                       file_name_layover: (
                        cfg.rdr2geo_params.compute_layover_shadow_mask,
                        gdal.GDT_Byte),
-                       'incidence': (rdr2geo_cfg.compute_incidence_angle,
-                                     gdal.GDT_Float32),
-                       'local_incidence': (
+                       file_name_local_incidence: (
                        rdr2geo_cfg.compute_local_incidence_angle,
                        gdal.GDT_Float32),
-                       'heading': (
-                       rdr2geo_cfg.compute_azimuth_angle, gdal.GDT_Float32)}
+                       file_name_los_east: (
+                       rdr2geo_cfg.compute_ground_to_sat_east, gdal.GDT_Float32),
+                       file_name_los_north: (
+                       rdr2geo_cfg.compute_ground_to_sat_north, gdal.GDT_Float32),
+                       }
         raster_list = [
             isce3.io.Raster(f'{output_path}/{fname}.rdr', rdr_grid.width,
                             rdr_grid.length, 1, dtype, 'ENVI')
             if enabled else None
             for fname, (enabled, dtype) in topo_output.items()]
 
         x_raster, y_raster, z_raster, layover_shadow_raster, \
-        incident_angle_raster, local_incident_angle_raster, \
-        azimuth_angle_raster = raster_list
+        local_incident_angle_raster, los_east_raster,\
+        los_north_raster = raster_list
```

Comment:
Just a style suggestion. You can keep the current style line splitting if you want.
```python
        (x_raster, y_raster, z_raster, layover_shadow_raster,
        local_incident_angle_raster, los_east_raster,
        los_north_raster) = raster_list
```